src/components/Navigation/NavigationItemTrigger/NavigationItemTrigger.tsx

```diff
@@ -45,15 +47,15 @@ const NavigationItemTrigger = forwardRef<HTMLLIElement, NavigationItemTriggerPro
 
     return (
       <>
-        <SSRMediaQuery.Media lessThan="md">
-          <MobileNavigationItemTrigger id={id} label={label} {...commonProps}>
-            {navListElement
-              ? React.cloneElement(navListElement[0], {
-                  className: `${baseClassName}__submenu--mobile`,
-                })
-              : undefined}
-          </MobileNavigationItemTrigger>
-        </SSRMediaQuery.Media>
+        {/* <SSRMediaQuery.Media lessThan="md"> */}
+        <MobileNavigationItemTrigger id={id} label={label} {...commonProps}>
+          {navListElement
+            ? React.cloneElement(navListElement[0], {
+                className: `${baseClassName}__submenu--mobile`,
+              })
+            : undefined}
+        </MobileNavigationItemTrigger>
+        {/* </SSRMediaQuery.Media> */}
```

Comment:
did this need to be removed?  Ideally it wouldn't render to the DOM for accessibility/screen readers